From b467385cd75a03b4aef07082fe758de847c2a5d7 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Wed, 29 Apr 2020 15:14:18 -0500
Subject: [PATCH] Update wallclock reftest expectations

Hebrew clock format was changed in 6e5e2cd513f908d21a69ca6ddad3c7431549efdc.

I've reported https://gitlab.gnome.org/Teams/Translation/he/-/issues/1
to confirm the change is intended.
---
 tests/he_IL.utf8.ref.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/he_IL.utf8.ref.ui b/tests/he_IL.utf8.ref.ui
index 75fd4cd7..576fa07a 100644
--- a/tests/he_IL.utf8.ref.ui
+++ b/tests/he_IL.utf8.ref.ui
@@ -9,7 +9,7 @@
       <object class="GtkLabel" id="label1">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="label" translatable="yes">ד' מאי 28 23‎∶59‎∶59</property>
+        <property name="label" translatable="yes">ד' ה־28 במאי 23‎∶59‎∶59</property>
       </object>
     </child>
   </object>
-- 
2.26.2

